example/android/app/src/main/java/com/example/MainApplication.java

```diff
@@ -25,8 +24,7 @@ protected boolean getUseDeveloperSupport() {
     @Override
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
-          new MainReactPackage(),
-          new RNSvgPackage()
```

Comment:
I tried out your changes on both iOS and Android and got errors on both

On iOS:
```
No component found for view with name "RNSVGGroup"
```

On Android:
```
No ViewManager defined for class RNSVGGroup
```

Doing:

```
react-native link react-native-svg
```

in the `example` dir fixes this problem and modifies the following 4 files:

```
git status

...
modified:   android/app/build.gradle
modified:   android/app/src/main/java/com/example/MainApplication.java
modified:   android/settings.gradle
modified:   ios/example.xcodeproj/project.pbxproj
```

The change that `react-native link ...` makes to `example/MainApplication.java` is the same one I mention above in "we probably will need to manually add back in the new RNSvgPackage() to make it work". So, no need to do the manual edit - we just need to do `react-native link react-native-svg` instead.

That should be all thats left to do to button this up.